docs/02-app/02-api-reference/01-components/image.mdx

```diff
@@ -725,6 +725,42 @@ Try it out:
 
 - [Demo the `fill` prop](https://image-component.nextjs.gallery/fill)
 
+## Theme Detection
+
+If you want to display a different image for light and dark mode, you can create a new component that wraps two `<Image>` components and reveals the correct one based on a CSS media query.
+
+```css filename="components/theme-image.module.css"
+.imgDark {
+  display: none;
+}
+
+@media (prefers-color-scheme: dark) {
+  .imgLight {
+    display: none;
+  }
+  .imgDark {
+    display: unset;
+  }
+}
+```
+
+```jsx filename="components/theme-image.js"
+import styles from './theme-image.module.css'
+const ThemeImage = (props) => {
+  const { srcLight, srcDark, ...rest } = props
+  return (
+    <>
+      <Image {...rest} src={srcLight} className={styles.imgLight} />
+      <Image {...rest} src={srcDark} className={styles.imgDark} />
+    </>
+  )
+}
+```
+
+> **Good to know**: This depends on the default behavior of `loading="lazy"` so that only the correct image is loaded. You cannot use `priority` or `loading="eager"` because it could cause both images to load.
```

Comment:
"Could" or "Would"? I think the latter right?

Reply:
```suggestion
> **Good to know**: This depends on the default behavior of `loading="lazy"` so that only the correct image is loaded. You cannot use `priority` or `loading="eager"` because that would cause both images to load.
```

Reply:
Done [09377db](https://github.com/vercel/next.js/pull/53760/commits/09377db5ef7139eb352c8da3892bed99905a14c1)